spec/lib/open_food_network/i18n_config_spec.rb

```diff
@@ -5,6 +5,12 @@
 
 module OpenFoodNetwork
   describe I18nConfig do
+    before do
+      allow(ENV).to receive(:[]).with("DATABASE_CLEANER_ALLOW_PRODUCTION").and_return("false")
+      allow(ENV).to receive(:[]).with("DATABASE_CLEANER_ALLOW_REMOTE_DATABASE_URL").
```

Comment:
This looks a bit weird to me. It seems like these should both be the default values anyway? https://github.com/DatabaseCleaner/database_cleaner#safeguards

If we want to set these configs, it feels cleaner to just do this in `spec_helper.rb`:
```ruby
DatabaseCleaner.allow_production = true
DatabaseCleaner.allow_remote_database_url = true
```

Reply:
this is actually coming from here: https://github.com/openfoodfoundation/openfoodnetwork/pull/7081/files
I just added it here to get the green build.

Reply:
Can we try dropping those lines that stub those two ENV vars and seeing what happens? I don't understand why they would be needed, as the defaults seem to be `false` for both of those options... :thinking:

Reply:
And it only breaks in this one spec file and not any others..?

Reply:
yes, only breaks here. I will add to spec_helper and see. I'll do that in the 7081 PR first

Reply:
Ah! We're also stubbing ENV elsewhere in this file. I guess it's messing with the return values when `DatabaseCleaner` tries to check `ENV[:DATABASE_CLEANER_ALLOW_PRODUCTION]` as a boolean, for example.

Reply:
Alright. Curiosity: satisfied.

In that case, I'm not sure it makes sense to explicitly set those values to their defaults in `spec_helper.rb`? The bug is a consequence of the way we're stubbing of `ENV`, and not by the actual test suite configs being incorrect...

Reply:
We could leave it as it is, but maybe add an explanatory comment for why it's needed in this specific file.

Reply:
ah, nice, thanks, makes sense! I did that in the PR https://github.com/openfoodfoundation/openfoodnetwork/pull/7081/files
It will come to this branch when rebased/merged from master :+1: